scripts/postinstall.js

```diff
@@ -40,19 +40,21 @@ try {
     }
   */
 
-	// if sites
-	// obj with key is truthy if sites property exists
-	Object.values(targetBlockValues.sites).forEach(({ siteProperties: sitePropertyObject }) => {
-		// if site has no date localization obj
-		if (sitePropertyObject.dateLocalization) {
-			if (typeof sitePropertyObject.dateLocalization.language === "string") {
-				themesLocaleList.push(sitePropertyObject.dateLocalization.language);
-			}
-			if (typeof sitePropertyObject.dateLocalization.timeZone === "string") {
-				targetTimeZones.push(sitePropertyObject.dateLocalization.timeZone);
+	// if sites exists
+	if (targetBlockValues.sites) {
+		// obj with key is truthy if sites property exists
+		Object.values(targetBlockValues.sites).forEach(({ siteProperties: sitePropertyObject }) => {
```

Comment:
Maybe I have an unhealthy aversion to if blocks, but I would have:
```suggestion
		Object.values(targetBlockValues.sites || []).forEach(({ siteProperties: sitePropertyObject }) => {
```

Reply:
no it's fair. 

But with the optional `?`, I avoided using this as fusion isn't using the supported node 14 yet. This is run on postinstall with that runtime. I'm not sure if I verified if the `logical or ||` is supported by 12. 

I'd rather not even get into the Object.values from a readability perspective. 

Thanks for reviewing.

Reply:
https://node.green/#ES2021-features-Logical-Assignment